src/modules/member/components/Add.js

```diff
@@ -0,0 +1,114 @@
+import React from 'react'
+import { Link } from 'react-router'
+
+import List from './List'
+
+import MemberService from '../services/MemberService'
+
+import { connect } from 'nuclear-js-react-addons'
+import moment from '../../../moment'
+import * as actions from '../actions'
+
+import Pagination from '../../../components/Pagination'
+import Loader from '../../../components/Loader'
+
+
+export default class Add extends React.Component{
+    constructor(props) {
+        super(props)
+
+        this.handleEmailChange = this.handleEmailChange.bind(this)
+        this.handleLifetimeChange = this.handleLifetimeChange.bind(this)
+        this.handleNameChange = this.handleNameChange.bind(this)
+        this.handleSubmit = this.handleSubmit.bind(this)
+
+        this.initState()
```

Comment:
Jeg ville nok hatt koden i initState her i stedet for siden den er så liten. Nå blir det egentlig bare tyngre å lese siden man må hoppe unødvendig. Men i og for seg helt greit, er mer en personlig preferanse